### DISCUSSION TITLE
Does skywalking support dubbo3?

### DISCUSSION THREAD

**Discussion by miaoxin** [apache/skywalking#8113] [Q&A] [answered]
I'm using skywalking java agent 8.8.0 in my dubbo3 project, but find exception below:
![image](https://user-images.githubusercontent.com/2310415/141327915-d8d2d7f1-e051-4eac-a289-a91e089bea7f.png)

**Comment by wu-sheng** [comment, marked as answer]
There are many issues discussed about this. Not supported, as no one contributed.
